### PR TITLE
Prs13881 irenaeus

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**

### CODE REVIEW

File: new/PRS13881Irenaeus.xml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            <change who="MKr" when="2022-08-28">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sameAs="Q240598" sex="1">
+                    <persName xml:lang="en" xml:id="n1">Irenaeus of Sirmium</persName>
+                    <persName xml:lang="gez" xml:id="n2">ሄሬኔዎስ፡ ጳጳስ፡ ዘሶርም፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n2">Herenewos ṗāṗṗās za-Sorǝm</persName>
```
Comment (thea-m) [inline new/PRS13881Irenaeus.xml:55]: same ṗāṗās question as in the work

**Review by thea-m — CHANGES_REQUESTED** [review]
Thanks a lot!

**Comment by MarcinKrawczuk** [comment]
Done!

wt., 30 sie 2022 o 11:38 thea-m ***@***.***> napisał(a):

**Review by thea-m — APPROVED** [review]
Thanks!

**Pull request merged**